litmus-portal/frontend/src/components/Sections/ChaosWorkflows/BrowseWorkflow/index.tsx

```diff
@@ -58,21 +61,23 @@ interface SortData {
 
 const BrowseWorkflow = () => {
   const classes = useStyles();
+  const userData: UserData = useSelector((state: RootState) => state.userData);
+  const { selectedProjectID } = userData;
 
   // Query to get workflows
   const { subscribeToMore, data, loading, error } = useQuery<
     Workflow,
     WorkflowDataVars
   >(WORKFLOW_DETAILS, {
-    variables: { projectID: '00002' },
+    variables: { projectID: selectedProjectID as string },
```

Comment:
is everything is typed correctly we do not need the as string type casting here...WorkflowDataVars already typed projectID as string